src/utilities/__tests__/buildASTSchema-test.js

```diff
@@ -32,61 +34,54 @@ function cycleOutput(body, options) {
 }
 
 describe('Schema Builder', () => {
-  it('can use built schema for limited execution', async () => {
+  it('can use built schema for limited execution', () => {
     const schema = buildASTSchema(
-      parse(`
-      schema { query: Query }
-      type Query {
-        str: String
-      }
-    `),
+      parse(dedent`
```

Comment:
I don't think we need to use `dedent` for these literals since we're not testing token locations nor doing string comparisons

Reply:
@leebyron It certain test failing because of buildSchema throw Error. It's easier to use location inside error to figure out what's wrong but without `dedent` location is not intuitive and hard to map on SDL string.